lib/scan-results.js

```diff
@@ -0,0 +1,61 @@
+
+const _ = require('lodash')
+
+const threshold = { low: 1, medium: 2, high: 3, critical: 4 }
+
+function flatMap (accumulator, chunk) {
```

Comment:
This is not necessary, we can use lodash’s flatmap functionality: https://lodash.com/docs/4.17.14#flatMap